quick-start.md

```diff
@@ -105,3 +107,196 @@ sudo su
+  * After you successfully run the script, all the configuration files have been downloaded and & stored into `<Installation Path>`
+  * If you want to update your app settings (ex: database host). Go to the `<Installation Path>/config-template`, update the corresponding value in the configmap file, then restart the pods.
+  * Below steps will help you update database hostname of your application:
+    * Open file `appsmith-configmap.yaml` in `<Installation Path>/config-template` folder
+    * Update the value of variable `APPSMITH_MONGODB_URI` to your database host name
+    * Run commands:
+```
+kubectl apply -f appsmith-configmap.yaml
+kubectl scale deployment appsmith-internal-server --replicas=0
+kubectl scale deployment appsmith-internal-server --replicas=1
+```
+
+{% hint style="success" %}
+* You can access the running application on the **Ingress Endpoint** if you not chose to provide custom domain for your application .
+```
+kubectl get ingress
+NAME               CLASS    HOSTS   ADDRESS          PORTS   AGE
+appsmith-ingress   <none>   *       XXX.XXX.XX.XXX   80      2m
+```
+* You may need to wait 2-3 minutes before accessing the application to allow application start (depends on your cluster).
+{% endhint %}
+
+### Custom Domains
+
+To host Appsmith on a custom domain, you can contact your domain registrar and update your DNS records. Most domain registrars have documentation on how you can do this yourself.
+
+* [GoDaddy](https://in.godaddy.com/help/create-a-subdomain-4080)
+* [Amazon Route 53](https://aws.amazon.com/premiumsupport/knowledge-center/create-subdomain-route-53/)
+* [Digital Ocean](https://www.digitalocean.com/docs/networking/dns/how-to/add-subdomain/)
+* [NameCheap](https://www.namecheap.com/support/knowledgebase/article.aspx/9776/2237/how-to-create-a-subdomain-for-my-domain)
+* [Domain.com](https://www.domain.com/help/article/domain-management-how-to-update-subdomains)
+
+{% hint style="warning" %}
+* During the setup of Ingress Controller on your cloud. You will need to map your custom domain with the External IP of the controller before running the installation script
+* Below is an example how to achieve the External IP of NGINX Ingress Controller
+```
+➜ kubectl get svc -n ingress-nginx
+NAME                                 TYPE           CLUSTER-IP    EXTERNAL-IP    PORT(S)                      AGE
+ingress-nginx-controller             LoadBalancer   XX.XXX.X.XX   XX.XX.XX.XXX   80:XXXXX/TCP,443:XXXXX/TCP   17h
+ingress-nginx-controller-admission   ClusterIP      XX.XXX.X.XX   <none>         443/TCP                      17h
+```
+{% endhint %}
+
+## Troubleshooting
+
+If at any time you encounter an error while installation Appsmith on any platform, reach out to **support@appsmith.com** or join our [Discord Server](https://discord.com/invite/rBTTVJp)
+
+If you know the error and would like to reinstall Appsmith, simply delete the installation folder and the templates folder and execute the script again
+
```

Comment:
Overall global comment -
1. I think we should run Grammarly on this to quickly fix typos, etc.
2. We need to ensure that every step works as expected on copy paste on all OS